flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connectors-doris/src/main/java/com/ververica/cdc/connectors/doris/sink/DorisDatabase.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.connectors.doris.sink;
+
+import com.ververica.cdc.common.event.AddColumnEvent;
+import com.ververica.cdc.common.event.AlterColumnTypeEvent;
+import com.ververica.cdc.common.event.CreateTableEvent;
+import com.ververica.cdc.common.event.DropColumnEvent;
+import com.ververica.cdc.common.event.RenameColumnEvent;
+import com.ververica.cdc.common.event.SchemaChangeEvent;
+import com.ververica.cdc.common.event.TableId;
+import com.ververica.cdc.common.schema.Column;
+import com.ververica.cdc.common.schema.Schema;
+import com.ververica.cdc.common.utils.Preconditions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.LinkedList;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class DorisDatabase {
+    private static final Logger LOG = LoggerFactory.getLogger(DorisDatabase.class);
+
+    public static final Map<TableId, DorisTable> globalTables = new ConcurrentHashMap<>();
+
+    public static void applySchemaChangeEvent(SchemaChangeEvent event) {
```

Comment:
this https://github.com/ververica/flink-cdc-connectors/pull/2732 pr provides a common method to apply SchemaChangeEvent.

Reply:
You can refer to [ValuesDataSink](https://github.com/ververica/flink-cdc-connectors/blob/b4deda97be942108371781c56858ff7d42650dc8/flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-values/src/main/java/com/ververica/cdc/connectors/values/sink/ValuesDataSink.java#L110) to maintain and update the schemas.